tests/jballerina-unit-test/src/test/resources/test-src/types/string/string_negative.bal

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2019 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Just for clarity, shall we rename this file to something like `string_multi_line_negative.bal`?